javascripts/discourse/connectors/above-latest-topic-list-item/clickable-topic-row.js

```diff
@@ -14,7 +14,7 @@ export default class extends Component {
       if (wantsNewWindow(event)) {
         return true;
       }
-      return navigateToTopic.call(this, topic, topic.lastUnreadUrl);
```

Comment:
Core's `LatestTopicListItem` doesn't have any special logic for navigation, so we can keep it simple here